src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -108,6 +108,11 @@ public static boolean isAuthHeaderPresent(Request req) {
         return authHeader != null;
     }
 
+    public static boolean isScopeHeaderPresent(Request req) {
+        final String scope = req.headers("scope");
```

Comment:
Non-blocking: There is some redundancy (this method is used only once, with a call to `req.headers("scope")` that immediately follows the call to this function. Similar remark applies to method `isAuthHeaderPresent` above.

Reply:
I've removed `isScopeHeaderPresent` and tidied up where is was called in `RequestingUser.createTestUser`. `isAuthHeaderPresent` is used in more than one place so have left this as is.